src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -2450,8 +2450,27 @@ private MemberDeclarationSyntax ParseMemberDeclarationOrStatement(SyntaxKind par
                     }
 
                     var incompleteMember = _syntaxFactory.IncompleteMember(attributes, modifiers.ToList(), type.IsMissing ? null : type);
+
+                    var containsErrorDiagnostic = false;
+                    // ContainsDiagnostics will be true if *any child node* contains 
+                    // a Diganostic, so only 
                     if (incompleteMember.ContainsDiagnostics)
                     {
+                        foreach (var child in incompleteMember.EnumerateNodes())
+                        {
+                            foreach (var diagnostic in child.GetDiagnostics())
```

Comment:
>var [](start = 37, length = 3)

This use of ```var``` is a violation of our policy. I cannot tell what are we dealing with here.